test/CMakeLists.txt

```diff
@@ -47,4 +47,5 @@ endforeach()
 
 target_compile_options(trace_fence PRIVATE -fsanitize=thread -Wno-error=tsan
                                            -Wno-unknown-warning-option)
+target_compile_options(trace_memcpy PRIVATE -fno-builtin)
```

Comment:
Just for my information, have you experienced issues without this flag?

Reply:
comment is good. If we don't have it, the memcpy is inlined and there is no instrumentation, as a result, the trace is not as expected. It's really just to force having a call to memcpy or __tsan_memcpy.